modules/gateway/rpc.go

```diff
@@ -61,7 +61,7 @@ func (g *Gateway) RPC(addr modules.NetAddress, name string, fn modules.RPCFunc)
 func (g *Gateway) RegisterRPC(name string, fn modules.RPCFunc) {
 	id := g.mu.Lock()
 	defer g.mu.Unlock(id)
-	if build.DEBUG {
+	if build.DEBUG && build.Release != "testing" {
```

Comment:
The rescan tests requires me to close and re-open the tpool a few times. The tpool calls RegisterRPC during `New`, causing panics because there's no way to disconnect an RPC from the gateway. I should drop a TODO here before this gets merged.

 Ideally, modules would be able to de-register their RPCs upon closing. Well, doubly important because the object is not going to be GC'd if the gateway is still pointing to it, and something might panic if you close a module but the gateway still has an RPC registered to it.